Jenkinsfile

```diff
@@ -0,0 +1,164 @@
+def registry = '562955126301.dkr.ecr.eu-west-2.amazonaws.com'
+def regCredsId = 'ecr:eu-west-2:ecr-user'
+def kubeCredsId = 'awskubeconfig002'
+def ingressServer = "ffc.aws-int.defra.cloud"
```

Comment:
Shouldn't need `ingressServer` for anything other than the web service.

Reply:
Indeed, removed.